.github/workflows/release.yaml

```diff
@@ -0,0 +1,30 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  publish:
+    name: Publish New Version to Pypi
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          persist-credentials: false
+      - name: Fetch pypi token from Akeyless
+        uses: LanceMcCarthy/akeyless-action@v4
```

Comment:
Shouldn't this be `LanceMcCarthy/akeyless-action@d42e07875b1cc7162afd290722babbaf879a86e6` notice this is version `5.0.3`